stories/Stepper/PalliniDark.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import StepperContent from '../../src/components/Stepper/StepperContent'
+import StepperNav from '../../src/components/Stepper/StepperNav'
+import { Button, Icon, StepperContainer } from '../../src'
+import StepperDots from '../../src/components/Stepper/StepperDots'
+
+const PalliniDark = () => {
+  return (
+    <StepperContainer mobile dark>
+      <StepperContent>
+        <p>Example content of the current step</p>
```

Comment:
You can add `className="text-white"` wherever you have black text on dark background. Although this shouldn't be done but as you have you StepperContent is different from that of BI, I think you should add this.

Reply:
But that doesn't solve the problem. It still doesn't match with that of BI. I think I should keep it as is so that user can decide for themselves. Let's wait for @dej611 's opinion on this.

Reply:
Yes, it doesn't match the styling of BI. Some issue with BI, I tried to replicate but it didn't work. Probably @francescozaia can have a look at this.

Reply:
Mind that the BI documentation has some special "documentation" styling on top of the regular one, like the height of the `StepperContent`, ref: https://github.com/italia/bootstrap-italia/blob/ac60866a0d06528931c526538d1fd89517a1e635/docs/componenti/steppers.md on the `<style>` tag on top.
For the Stepper in particular you can copy the same Content styling and use it into the stories, as long as it is visible in the `Info` panel.

Reply:
This PR has already got long, why not add a `callout` or `card` component at the moment? I guess we can fix this in some other issue?

Reply:
I added the styles @dej611 had directed me to. It seems to work just fine 😃

Reply:
The new changes look good 👍

Reply:
One last change 🐔